### ISSUE TITLE
ec2_vol reports changed = false even if a new vol is created or attched

### ISSUE THREAD

**Issue by bennojoy** [ansible/ansible-modules-core#1308]
##### Issue Type:

Bug Report
##### Component Name:

ec2_vol 
##### Summary:

ec2_vol reports changed = false even if a new vol is created or attched
##### Ansible Version:

1.9 +
##### Environment:

N/A
##### Steps To Reproduce:

The Playbook to create a new volume and attach to a instance

```
- hosts: localhost
  tasks:
    - ec2_vol:
        instance: i-12345
        volume_size: 3
        name: benz
        zone: "us-east-1c"
        device_name: xvdq
```
##### Expected Results:

```
Changed=True
```
##### Actual Results:

```
{"changed": false, "device": "xvdq", "volume_id": "vol-816b9d6f", "volume_type": "standard"}
```

**Comment by wimnat** [comment]
Duplicate of https://github.com/ansible/ansible-modules-core/issues/695.

Fix proposed: https://github.com/ansible/ansible-modules-core/pull/2180

**Comment by ansibot** [comment]
@ansible, ping. This issue is waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by wimnat** [comment]
bug_resolved
resolved_by_pr https://github.com/ansible/ansible-modules-core/pull/2180

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by wimnat** [comment]
@bot_broken this can be closed using tags above

**Comment by ansibot** [comment]
This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.
